DistributionModels/weibull.py

```diff
@@ -2,14 +2,18 @@
 import torch
 
 class weibull:
-    def __init__(self, saved_model=None):
+    def __init__(self, translateAmountTensor=1,saved_model=None):
+        self.smallScoreTensor = 0.0
+        self.reversed = False
+        self.translateAmountTensor = translateAmountTensor
         if saved_model:
             self.wbFits = torch.zeros(saved_model['Scale'].shape[0],2)
             self.wbFits[:, 1] = saved_model['Scale']
             self.wbFits[:, 0] = saved_model['Shape']
             self.sign = saved_model['signTensor']
             self._ = saved_model['translateAmountTensor']
```

Comment:
This line should be removed since it was meant to ignore `translateAmountTensor`.